client/src/components/Layout/Header.tsx

```diff
@@ -79,51 +76,47 @@ const Header: React.FC = () => {
           <img className="logo" alt={title} src="/assets/logos/cqgc-white.svg" />
         </Col>
         <div className="secondaryNav">
-          { user.username !== null && (
+          {user.username !== null && (
```

Comment:
(Optional) not your code but `!== null` is probably not exactly what is needed. I guess it would be more something like `user.username && ...`, but I m not sure.

Reply:
Ye, i did a new component from scratch in clin2, but ill fix it here anyway